.github/workflows/workflow-integration-tests.yaml

```diff
@@ -0,0 +1,311 @@
+name: Workflow Integration Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Select the Cassandra or DSE version to run tests against'
+        required: true
+        default: '4.0'
+        type: choice
+        options:
+          - '3.11'
+          - '4.0'
+          - '6.8'
+          - 'ALL'
+
+jobs:
+  build_docker_images:
+    name: Build Docker Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v3
+        with:
+          file: Dockerfile
+          context: .
+          push: false
+          tags: k8ssandra/cass-operator:latest
+          platforms: linux/amd64
+          outputs: type=docker,dest=/tmp/k8ssandra-cass-operator.tar
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          file: logger.Dockerfile
+          push: false
+          tags: k8ssandra/system-logger:latest
+          outputs: type=docker,dest=/tmp/k8ssandra-system-logger.tar
+          platforms: linux/amd64
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache
+      - name: Upload cass-operator image
+        uses: actions/upload-artifact@v3
+        with:
+          name: cass-operator
+          path: /tmp/k8ssandra-cass-operator.tar
+      - name: Upload system-logger image
+        uses: actions/upload-artifact@v3
+        with:
+          name: system-logger
+          path: /tmp/k8ssandra-system-logger.tar
+  kind_311_tests:
+    if: ${{ inputs.version == '3.11' || inputs.version == 'ALL' }}
+    needs: build_docker_images
+    strategy:
+      matrix:
```

Comment:
Is this missing version parameter?

Reply:
Yes. It might not need it right now as the tests in this matrix might all be using Cassandra 3.11 specs, but setting the version explicitly will ensure it. Fix coming